VideoLocker/build.gradle

```diff
@@ -96,6 +96,7 @@ dependencies {
     compile 'com.android.support:cardview-v7:21.0.0'
     compile 'com.android.support:appcompat-v7:22.1.0'
     compile 'com.android.support:multidex:1.0.0'
+    compile 'com.github.bumptech.glide:glide:3.6.1'
```

Comment:
Since we're using Glide as our image loader now, we can get rid of Volley and it's associated helper classes.